spring-cloud-gcp-docs/src/main/asciidoc/config.adoc

```diff
@@ -0,0 +1,117 @@
+== Spring Cloud Config
+
+Spring Cloud GCP makes it possible to use the
+https://cloud.google.com/deployment-manager/runtime-configurator/reference/rest/[Google
+Runtime Configuration API] as a
+https://cloud.spring.io/spring-cloud-config/[Spring Cloud Config] server to remotely store your
+application configuration data.
+
+The Spring Cloud GCP Config support is provided via a Spring Boot starter that makes notation like
+`@Value("${gcp_runtime_config_api_var_name}")` populate variables with values stored in the
+Google Runtime Configuration API.
+
+Maven coordinates, using Spring Cloud GCP BOM:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-gcp-starter-config</artifactId>
+</dependency>
+----
+
+Gradle coordinates:
+
+[source,subs="normal"]
+----
+dependencies {
+    compile group: 'org.springframework.cloud', name: 'spring-cloud-gcp-starter-config', version: '{project-version}'
+}
+----
+
+=== Configuration
+
+The following parameters are configurable in Spring Cloud GCP Config:
+
+|===
+| Name | Description | Optional | Default value
+| `spring.cloud.gcp.config.enabled` | Enables the Config client | Yes | `true`
+| `spring.cloud.gcp.config.name` |
```

Comment:
Is it config name or application name? It's confusing because you're linking to docs about `spring.application.name`. Should we just use `spring.application.name` or at least default to it?

Reply:
They're not related. It's suggested that this is set to `spring.application.name`, however, that's merely a suggestion, users can name the config anything they want.

Reply:
I think we can omit suggesting any connection between this and `spring.application.name` and also not link to http://cloud.spring.io/spring-cloud-static/spring-cloud-config/1.3.3.RELEASE/single/spring-cloud-config.html#_locating_remote_configuration_resources.